core/storage/user/gae_models.py

```diff
@@ -2142,10 +2144,16 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     # IDs of all the private collections created by this user.
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
 
-    # A dict mapping story IDs to pseudonymous user IDs. For each story, we
-    # use a different pseudonymous user ID. Note that all these pseudonymous
+    # A dict mapping model IDs to pseudonymous user IDs. Each type of activity
+    # is grouped under different key (story, skill, question), the keys need to
+    # be from the core.platform.models.NAMES enum. For each activity, we use
+    # a different pseudonymous user ID. Note that all these pseudonymous
     # user IDs originate from the same about-to-be-deleted user.
-    story_mappings = ndb.JsonProperty(default={})
+    # Example structure: {
+    #   'skill': {'skill_id': 'pseudo_user_id_1'},
+    #   'story': {'story_id': 'pseudo_user_id_2'}
```

Comment:
add 'question' (and examples where there are 0 of a kind, 2+ of a kind)?

Reply:
Done.